ubidots/ubidots_in.html

```diff
@@ -1,111 +1,608 @@
+          <input type="checkbox" id="ubidots-checkbox-variable-10" style="display: inline-block; margin-right:10px; width: auto; vertical-align: center;">
+          <input type="text" style="width: calc(100% - 250px);" id="node-input-label_variable_10" />
+          <input type="checkbox" checked id="node-input-checkbox_variable_10_last_value" style="display: inline-block; margin-right:10px; margin-left:15px; width: auto; vertical-align: center;">
+          <label id="node-input-last-value-label-10" style="width: 89px" data-i18n="ubidots.lastValue"></label>
+          <button class="red-ui-button" id="ubidots-variable-button-key-clear-10" style="margin-left: 15px"><i style="color:#525050;" class="fa fa-times"></i></button>
+      </div>
+      <hr>
+      <!-- Add Button -->
+      <div id="add-button" class="form-row" style="margin-top:10px">
+          <label style="width: calc(30%)" id="ubidots-add-variable-button-label" for="ubidots-add-variable-button">
+          </label>
+          <span  style="width: calc(15%)">
+         <button class="red-ui-button" id="ubidots-add-variable-button"><i class="fa fa-plus"></i></button>
+      </span>
 
+      </div>
+</script>
 
 <script type="text/x-red" data-help-name="ubidots_in">
-    <p>Fetch the last value from Ubidots and send it as a message to other nodes.</p>
-
-    <p>
-    Uses the <a href="https://github.com/mqttjs/MQTT.js" target="_blank">MQTT</a> to make the connection and suscribe
-    to the channel.
-    </p>
-
-    <p>
-    You must configure your node by double clicking it and filling the following information, all fields are required:
-    <br />
-
-    <ul>
-        <li>
-            <b>Account type</b> - The type of your account, whether it is a Ubidots or Ubidots for
-            Education account.
-        </li>
-        <li>
-            <b>Name</b> - The name to identify your MQTT client.
-        </li>
-        <li>
-            <b>Token</b> - Your Ubidots' account token. You can find it by following
-            <a href="http://help.ubidots.com/user-guides/find-your-token-from-your-ubidots-account" target="_blank">
-                this tutorial
-            </a>.
-        </li>
-        <li>
-            <b>Device label</b> - The label of the Device that contains the Variable where you want to fetch the values.
-        </li>
-        <li>
-            <b>Variable label</b> - The label of the Variable that you want to fetch the values from.
-        </li>
-    </ul>
-    </p>
-
-    <p>
-    Refer to the <a href="https://ubidots.com/docs/api/mqtt.html" target="_blank">Ubidots documentation</a> for more
-    information about the functionality of the MQTT broker.
-    </p>
+  <p>Subscribes to up to 10 variables or topics from Ubidots and sends them as a message to other nodes.</p>
+
+  <p>
+      Uses the <a href="https://github.com/mqttjs/MQTT.js" target="_blank">MQTT</a> library to establish the connection and suscribe to the topic.
```

Comment:
Set a `rel="noopener noreferrer"` property to prevent cross-sites from using `window.opener` property and isolating the process.